modules/UI/side_pannels/profile/Profile.js

```diff
@@ -54,6 +53,9 @@ export default {
     init(emitter) {
         initHTML();
 
+        const profile
+            = APP.store ? APP.store.getState()['features/base/profile'] : {};
```

Comment:
Which case happens more, that it's defined or that it's not? Do you know why this happens in general? The flow `Conference.web#componentDidMount -> UI#start -> SidePanels#init -> Profile#init` make me think APP.store should always be available because it happens after the react Conference has mounted.

Reply:
Very true, thanks!